src/presentation/pages/Home.vue

```diff
@@ -1,7 +1,20 @@
 <template>
-  <div>
-    <h1>Recent Notes</h1>
-    <p>This page will contain your Notes you recently worked with </p>
+  <div class="page__block">
```

Comment:
1. If you're using BEM — you're declared an element, but the element should be included in a Block. 
2. It's better to place styles for blocks and elements inside their component. See [Colocation of inherently coupled concerns](https://vuejs.org/#what-about-separation-of-concerns).  
3. Styles should be applied to the Note (with editor), not a static page (Home)
4. If you want to move note styles to the `typography.pcss` to allow use them somewhere else except the editor (where?), maybe it would be better to use CSS Modules instead of bem? Because for now styles are coupled with their classes, so we will need to use the same classes (`.page__title--2`) in different components?